data/remnant.txt

```diff
@@ -248,14 +333,25 @@ phrase "contacted remnant hostile hail"
 	word
 		". "
 	word
-		"You must be "
+		"You"
 	word
-		"destroyed."
-		"killed."
+		" "
+	word
+		"should be"
+		"must be"
+	word
+		"destroyed"
+		"killed"
+		"out of our space"
+		"gone"
 
 phrase "contacted remnant hostile disabled hail"
 	word
-		"Human, if you "
+		"Human,"
+		"Brother,"
```

Comment:
the game never assumes the players' gender, so this one shouldn't exist as it assumes a male